src/Compilers/Test/Core/Compilation/TestOperationVisitor.cs

```diff
@@ -768,6 +772,10 @@ public override void VisitBinaryOperator(IBinaryOperation operation)
             }
 
             AssertEx.Equal(new[] { operation.LeftOperand, operation.RightOperand }, operation.ChildOperations);
+
+            // Directly get the symbol for this operator from the semantic model.  This allows us to exercise
+            // potentially creating synthesized intrinsic operators.
+            operation.SemanticModel?.GetSymbolInfo(operation.Syntax);
```

Comment:
>operation.SemanticModel?.GetSymbolInfo(operation.Syntax); [](http://example.com/codeflow?start=12&length=57)

I though the intent was to not just get the symbol, but also recreate it if it is built-in.